src/odemis/acq/stitching/_weaver.py

```diff
@@ -153,7 +166,7 @@ def get_bounding_boxes(tiles: list):
         if numpy.greater(gbbx_px[-2:], 4 * numpy.sum(tbbx_px[-2:])).any():
             # Overlap > 50% or missing tiles
             logging.warning("Global area much bigger than sum of tile areas")
-        return tbbx_px, gbbx_px, gbbx_phy
+        return tbbx_px, gbbx_px, gbbx_phy, mean_stage_bare_pos
```

Comment:
Please update the docinfo with a description for this extra return. Especially it's helpful to "remind" that the STAGE_POSITION_RAW can contain more than 2 axes (eg, 5).